index.html

```diff
@@ -55,6 +55,7 @@ <h1 class="text-white text-center">Sign in</h1>
 
     <p class="bg-white text-center">
       The code source of this page is available on <a href="https://github.com/simple-login/simplelogin-demo">Github</a>
+      please goto github and make some changes and improve this website
```

Comment:
I think we don't have to explicitly state that as anyone who wants to contribute will click on the GitHub link :)